designSystem/atoms/SelectRadix.tsx

```diff
@@ -81,17 +81,19 @@ const renderGroupedItems = (itemOptions: Array<GroupItemOptions>) => {
   )
 }
 
+// MAIN COMPONENT
 const SelectRadix: FC<SelectRadixProps> = forwardRef<
```

Comment:
> Sure, your form controller (select or whatever) must be wrapped in forwardRef and in cases like Select you have to map onChange to onValueChange.
> 
> This is bridging the API if a custom component to the Controller's.
> 
> But I see something weird with the way your using forwardRef, can you paste here the PR line so I can review there?
> 
> That setRefSelect looks weird.

@asherccohen here it is

Reply:
I tried to follow the pattern in the react docs. Not sure If I'm missing something but seems to align with it:
https://react.dev/reference/react/forwardRef 
```
// From react docs
const MyInput = forwardRef(function MyInput(props, ref) {
  return (
    <label>
      {props.label}
      <input ref={ref} />
    </label>
  );
})
```

Only other difference I could see is I destructure the props

Reply:
False alarm. I figured out the problem. I didn't realize I need to include each input in the zod schema, regardless if the input has validation rules. 

This fixed the problem and SelectRadix now works with Field.Control as expected

Reply:
If you still see something odd with the way I'm using `forwardRef` or passing RHF props to `SelectRadix` it would be great to know as I'll likely apply a similar pattern to other atomic components that use Radix UI primitives. Thanks